### DISCUSSION TITLE
Need help with looping between multiple images

### DISCUSSION THREAD

**Discussion by lewlite** [Nain57/Smart-AutoClicker#352] [Q&A]
I had several farming and building game where it could help with other player's farm. In 1 game there is multiple help icon image at same screen and when clicked it will show the confirmation button and the farm type image on it and every player have their help icon on different position as how they put the building. And only 1 help is allowed for each player so after 1 help icon clicked and confirmed, all the rest of help icon will disappeared.

The issue is when I put a condition for the farm type image and the condition was not found, it will close the confirmation window but the app will keep clicking on help icon at same position. I tried to swipe if the condition not found but if the player only had small farm, after the swipe still showing the same screen. I did use the timeout so it will confirmed anyway or else it became infinite loop of clicking and closing the same help icon.

But is there a way to mark the image position so it won't click on it again?

**Comment by Cyber5imon** [comment]
You can enable/disable scenarios which would allow you to click on an item and then disable the scenerio so it won't click twice.

**Reply by lewlite** [reply]
I guess yeah it could work that way too
It's a bit hard to set it since there is many conditions I had
But I saw Nain57 shared the a screenshot of the new layout for enabling and disabling scenarios
Can't wait to try those even though the app update for Play Store is a bit slower
